### PR TITLE
Added a section about Transformations

### PR DESCRIPTION
**Pull request by otisg**
Just a bit of info we can point to from the new AR create screen.

### CODE REVIEW

File: docs/alerts/creating-logs-alerts.md [hunk 1]
```diff
@@ -30,6 +30,18 @@ If you wanted to avoid using a specific threshold value and instead get notified
...
+
+```
+request.size + response.size
+```
+
+And then you would create an alert on this new data series.
+
+Read about [Transformations](../dashboards/chart-builder/#transformation) to leadn more about transformations, functions, and expressions used to perform transformations.
+
 Notice that the chart changed to help you get a sense of what would constitute an anomalous value in your case (dots outside the gray *confidence interval*).
```
Comment (emir-arnautovic) [inline docs/alerts/creating-logs-alerts.md:45]: Should this go before transformation section - it is referring to the image above?

File: docs/alerts/creating-metrics-alerts.md [hunk 2]
```diff
@@ -34,6 +34,19 @@ If you don't want to use a specific threshold value and instead get notified whe
...
+
+Transformations are used to modify metrics using mathematical expressions and functions. For example, imagine you want to get alerted when both `os.cpu.user` and `os.cpu.system` together exceed some threshold or become anomalous.  You would then transform them into a single dataseries by using an expression like this:
+
+```
+os.cpu.user + os.cpu.system
+```
+
+And then you would create an alert on this new data series.
+
+Read about [Transformations](../dashboards/chart-builder/#transformation) to leadn more about transformations, functions, and expressions used to perform transformations.
```
Comment (emir-arnautovic) [inline docs/alerts/creating-metrics-alerts.md:47]: The same comments as for logs.

**Review by emir-arnautovic — APPROVED** [review]
Added section looks good, but I think it should be moved a but further down.

**Comment by otisg** [comment]
Moved it/them down to the bottom.

**Pull request merged**